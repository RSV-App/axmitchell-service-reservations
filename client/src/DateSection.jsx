import React from 'react';
import PropTypes from 'prop-types';

const DateSection = (props) => {
  return (
    <div className="date-wrapper">
      <div className="date-title">Date</div>
      <div className="display-date" onClick={props.handleClick} >
        {props.clickedDate}
      </div>
    </div>
  );
};

DateSection.propTypes = {
  handleClick: PropTypes.func,
  clickedDate: PropTypes.string,
};

export default DateSection;
