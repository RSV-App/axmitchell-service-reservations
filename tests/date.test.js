import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import DateSection from '../client/src/DateSection.jsx';

configure({ adapter: new Adapter() });

it('renders correctly', () => {
  const tree = renderer
    .create(<DateSection />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
