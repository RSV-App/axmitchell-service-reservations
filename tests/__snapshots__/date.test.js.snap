// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="date-wrapper"
>
  <div
    className="date-title"
  >
    Date
  </div>
  <div
    className="display-date"
  />
</div>
`;
